template/package.json

```diff
@@ -28,6 +30,8 @@
     "react-native": "0.70.4",
     "react-native-code-push": "^7.0.5",
     "react-native-config": "^1.4.11",
+    "react-native-device-info": "^10.3.0",
+    "react-native-draggable": "^3.3.0",
```

Comment:
We can use panResponders instead of `react-native-draggable`

Reply:
Do we create another ticket for this? we would like to merge localization first